kentikapi/internal/httputil/httputil.go

```diff
@@ -25,16 +25,12 @@ func NewRetryingClient(cfg ClientConfig) *retryablehttp.Client {
 
 	c := retryablehttp.NewClient()
 	c.HTTPClient = cfg.HTTPClient
-	if cfg.RetryCfg.MaxAttempts != nil {
-		c.RetryMax = *cfg.RetryCfg.MaxAttempts
-	}
-	if cfg.RetryCfg.MinDelay != nil {
-		c.RetryWaitMin = *cfg.RetryCfg.MinDelay
-	}
-	if cfg.RetryCfg.MaxDelay != nil {
-		c.RetryWaitMax = *cfg.RetryCfg.MaxDelay
-	}
-	c.CheckRetry = makeRetryPolicy(cfg.RetryCfg.RetryableStatusCodes, cfg.RetryCfg.RetryableMethods)
+
```

Comment:
I would sleep better if you wouldn't remove that nil checks. There is always risk that pointers are uninitialized, as the code evolves

Reply:
Done